src/group_theory/subgroup.lean

```diff
@@ -1150,7 +1183,10 @@ instance decidable_mem_ker [decidable_eq N] (f : G →* N) :
 @[to_additive]
 lemma comap_ker (g : N →* P) (f : G →* N) : g.ker.comap f = (g.comp f).ker := rfl
 
-@[to_additive] lemma range_restrict_ker (f : G →* N) : ker (range_restrict f) = ker f :=
+@[to_additive] lemma comap_bot (f : G →* N) :
+  (⊥ : subgroup N).comap f = f.ker := rfl
+
```

Comment:
I'm not convinced this lemma is useful - I think `rw ←ker` does the same thing as `rw comap_bot`.

Reply:
I think it is useful for exactly the same reasons I think `map_top` is useful, as [discussed on Zulip](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/.60linear_map.2Erange_eq_map.60.20vs.20.60submodule.2Emap_top.60).

Reply:
Based on the zulip discussion shouldn't this also be `simp`?

Reply:
```suggestion
@[simp, to_additive] lemma comap_bot (f : G →* N) :
  (⊥ : subgroup N).comap f = f.ker := rfl

```